src/SelfServe/SelfServeUtils.tsx

```diff
@@ -17,29 +27,40 @@ export enum SelfServeType {
   invalid = "invalid",
   // Add your self serve types here
   example = "example",
+  sqlx = "sqlx",
```

Comment:
One idea would be to define the module that gets in `SelfServeComponentAdapter.getDescriptor()` in the URL, so that we avoid defining a type for each application and the `switch()` in `getDescriptor()`. (This said, it might open the door to dynamically loading any code, so this might be the safest).